compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -619,6 +661,70 @@ class ComposeScene internal constructor(
     @ExperimentalComposeUiApi
     fun moveFocus(focusDirection: FocusDirection): Boolean =
         list.lastOrNull()?.focusManager?.moveFocus(focusDirection) ?: false
+
+    /**
+     * Represents pointer such as mouse cursor, or touch/stylus press.
+     * There can be multiple pointers on the screen at the same time.
+     */
+    @ExperimentalComposeUiApi
+    class Pointer(
+        /**
+         * Unique id associated with the pointer. Used to distinguish between multiple pointers that can exist
+         * at the same time (i.e. multiple pressed touches).
+         */
+        val id: PointerId,
+
+        /**
+         * The [Offset] of the pointer.
```

Comment:
relative to what?

Reply:
The position inside `Pointer` can be relative to anything, depending where it is used. When we use it as a parameter of `sendPointerEvent`, it is mentioned, relative to what it should be:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/5963351/228788952-ee696e8c-51d0-4936-b94b-b0f83d48772e.png">